torch/distributions/distribution.py

```diff
@@ -260,7 +260,7 @@ def _get_checked_instance(self, cls, _instance=None):
 
     def __repr__(self):
         param_names = [k for k, _ in self.arg_constraints.items() if k in self.__dict__]
-        args_string = ', '.join(['{}: {}'.format(p, self.__dict__[p]
-                                if self.__dict__[p].dim() == 0
+        args_string = ', '.join(['{}: {}'.format(p, self.__dict__[p].item()
```

Comment:
I'm ambivalent about the requirement of `.item()` here.

Reply:
I agree. I went ahead and axed it.